### PR TITLE
DO-904 Added Privacy policy link

### PR DESCRIPTION
**Pull request by Litsyshyn**
by the task request https://razom.atlassian.net/browse/DO-904, added  Privacy policy link to the Sign-Up page.
Corrected issue that only EN version opens.

Also added back function to "Back"  Privacy Policy, according to https://razom.atlassian.net/browse/DO-920 task. 
Finding out the necesserity of this task is in progress

### CODE REVIEW

File: src/components/pages/auth/SignUpForm.jsx [hunk 1]
```diff
@@ -188,10 +193,12 @@ const SignUpForm = () => {
             checked={formik.values.accept_policy}
             onChange={formik.handleChange}
           />
-          <label className="cursor-pointer select-none" htmlFor="accept_policy">
+          <label className="cursor-pointer select-none mr-2" htmlFor="accept_policy">
             {t('iAgreeWith')}
           </label>
-          <a className="text-theme-1 ml-1" href="/docs/PrivacyPolicyEn.html" target="_blank">{t('privacyPolicy')}</a>.
+          <a className="text-theme-1" href={` ${process.env.PUBLIC_URL}${policy}`} target="_blank">
```
Comment (roman-tiukh) [inline src/components/pages/auth/SignUpForm.jsx:199]: extra space
```suggestion
          <a className="text-theme-1" href={`${process.env.PUBLIC_URL}${policy}`} target="_blank">
```

File: public/docs/PrivacyPolicyEn.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
         <h1 class="font-sans flex-1 text-left md:text-xl font-extrabold uppercase">
             Privacy policy
         </h1>
-        <a class="items-center button flex-4 w-24 mr-1 border text-white dark:border-dark-5" href="https://dp.dataocean.us/?lang=en">
+        <a class="items-center button flex-4 w-24 mr-1 border text-white dark:border-dark-5" href="javascript:history.back()">
```
Comment (roman-tiukh) [inline public/docs/PrivacyPolicyEn.html:14]: better use button with onclick event

File: public/docs/PrivacyPolicyUk.html [hunk 3]
```diff
@@ -11,7 +11,7 @@
         <h1 class="font-sans flex-1 md:text-xl font-extrabold uppercase">
             ПОЛІТИКА КОНФІДЕНЦІЙНОСТІ 
         </h1>
-        <a class="button flex-4 w-32 mr-1 border text-white dark:border-dark-5" href="https://dp.dataocean.us/?lang=uk">
+        <a class="button flex-4 w-32 mr-1 border text-white dark:border-dark-5" href="javascript:history.back()">
```
Comment (roman-tiukh) [inline public/docs/PrivacyPolicyUk.html:14]: better use button with onclick event

File: src/components/pages/auth/SignUpForm.jsx [hunk 4]
```diff
@@ -188,10 +193,12 @@ const SignUpForm = () => {
             checked={formik.values.accept_policy}
             onChange={formik.handleChange}
           />
-          <label className="cursor-pointer select-none" htmlFor="accept_policy">
+          <label className="cursor-pointer select-none mr-2" htmlFor="accept_policy">
```
Comment (roman-tiukh) [inline src/components/pages/auth/SignUpForm.jsx:196]: mr-2? i dont see problems with margins
image

File: src/components/pages/auth/SignUpForm.jsx [hunk 5]
```diff
@@ -188,10 +193,12 @@ const SignUpForm = () => {
...
             {t('iAgreeWith')}
           </label>
-          <a className="text-theme-1 ml-1" href="/docs/PrivacyPolicyEn.html" target="_blank">{t('privacyPolicy')}</a>.
+          <a className="text-theme-1" href={` ${process.env.PUBLIC_URL}${policy}`} target="_blank">
+            {t('privacyPolicy')}
+          </a>
```
Comment (roman-tiukh) [inline src/components/pages/auth/SignUpForm.jsx:201]: you missed dot
```suggestion
          </a>.
```

**Comment by Litsyshyn** [comment]
Updated

**Comment by Litsyshyn** [comment]
Sorry, do not did everything. 
I'll ask for request later.

File: src/components/pages/auth/SignUpForm.jsx (hunk 4 above)
Comment (Litsyshyn) [reply src/components/pages/auth/SignUpForm.jsx:196]: Witout mr-2, there is no space between "I agree with" and "Privacy policy" and looks like "I agree withPrivacy policy"

File: src/components/pages/auth/SignUpForm.jsx (hunk 5 above)
Comment (Litsyshyn) [reply src/components/pages/auth/SignUpForm.jsx:201]: i delete it purposely,  I thought that is mistake ))))

File: src/components/pages/auth/SignUpForm.jsx [hunk 6]
```diff
@@ -188,10 +193,12 @@ const SignUpForm = () => {
             checked={formik.values.accept_policy}
             onChange={formik.handleChange}
           />
-          <label className="cursor-pointer select-none" htmlFor="accept_policy">
+          <label className="cursor-pointer select-none mr-2" htmlFor="accept_policy">
             {t('iAgreeWith')}
           </label>
-          <a className="text-theme-1 ml-1" href="/docs/PrivacyPolicyEn.html" target="_blank">{t('privacyPolicy')}</a>.
+          <a className="text-theme-1" href={`${process.env.PUBLIC_URL}${policy}`} target="_blank">
```
Comment (roman-tiukh) [inline src/components/pages/auth/SignUpForm.jsx:199]: you delete ml-1..

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: src/components/pages/auth/SignUpForm.jsx (hunk 6 above)
Comment (Litsyshyn) [reply src/components/pages/auth/SignUpForm.jsx]: fixed
commited

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**